### PR TITLE
Scheduled posts are now listed and can be deleted

### PR DESCRIPTION
**Pull request by annando**
a very basic frontend is added to be able to see and delete scheduled posts.

### CODE REVIEW

File: src/Module/Profile/Schedule.php [hunk 1]
```diff
@@ -33,11 +38,46 @@ public static function content(array $parameters = [])
 			throw new HTTPException\ForbiddenException(DI::l10n()->t('Permission denied.'));
 		}
 
+		if (!empty($parameters['id'])) {
+			self::deleteSchedule($parameters['id']);
```
Comment (MrPetovan) [inline src/Module/Profile/Schedule.php]: This should be in the `post()` section.

File: view/theme/frio/templates/profile/schedule.tpl [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+				<tr>
+					<td>{{$row.scheduled_at}}</td>
+					<td>{{$row.content}}</td>
+					<td><a href="{{$baseurl}}/profile/{{$nickname}}/schedule/delete/{{$row.id}}?t={{$form_security_token}}" class="btn" title="{{$delete}}"><i class="fa fa-trash" aria-hidden="true"></i></a></td>
+				</tr>
+				{{/foreach}}
+			</tbody>
+		</table>
+	</form>
+</div>
```
Comment (MrPetovan) [inline view/theme/frio/templates/profile/schedule.tpl]: You can use this template as is for the base template. No need for different theme versions.

File: view/theme/frio/templates/profile/schedule.tpl [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+					<th>{{$scheduled_at}}</th>
+					<th>{{$content}}</th>
+				</tr>
+			</thead>
+			<tbody>
+				{{foreach $schedule as $row}}
+				<tr>
+					<td>{{$row.scheduled_at}}</td>
+					<td>{{$row.content}}</td>
+					<td><a href="{{$baseurl}}/profile/{{$nickname}}/schedule/delete/{{$row.id}}?t={{$form_security_token}}" class="btn" title="{{$delete}}"><i class="fa fa-trash" aria-hidden="true"></i></a></td>
```
Comment (MrPetovan) [inline view/theme/frio/templates/profile/schedule.tpl]: Please replace this link with a form with a security token and a submit button.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: src/Module/Profile/Schedule.php (hunk 1 above)
Comment (annando) [reply src/Module/Profile/Schedule.php]: The delete is not transmitted as `post`.

File: view/theme/frio/templates/profile/schedule.tpl (hunk 2 above)
Comment (annando) [reply view/theme/frio/templates/profile/schedule.tpl]: There are fine differences in - for example - the header size. I copied it from the oauth list.

File: view/theme/frio/templates/profile/schedule.tpl (hunk 3 above)
Comment (annando) [reply view/theme/frio/templates/profile/schedule.tpl]: Du you have an example for this? At several places (oauth, users) we seem to make it via some `get` action.

File: view/theme/frio/templates/profile/schedule.tpl (hunk 2 above)
Comment (MrPetovan) [reply view/theme/frio/templates/profile/schedule.tpl]: I don't think the differences are meaningful enough to warrant maintaining two different templates.

File: view/theme/frio/templates/profile/schedule.tpl (hunk 3 above)
Comment (MrPetovan) [reply view/theme/frio/templates/profile/schedule.tpl]: In this case:
```suggestion
					<td>
						<form action="{{$baseurl}}/profile/{{$nickname}}/schedule" method="post">
							<input type="hidden" name="form_security_token" value="{{$form_security_token}}">
							<button type="submit" name="delete" value="{{$row.id}}" title="{{$delete}}">
								<i class="fa fa-trash" aria-hidden="true">
							</button>
						</form>
					</td>
```

File: view/theme/frio/templates/profile/schedule.tpl (hunk 2 above)
Comment (annando) [reply view/theme/frio/templates/profile/schedule.tpl]: It looks inconsistent.

File: view/theme/frio/templates/profile/schedule.tpl (hunk 3 above)
Comment (annando) [reply view/theme/frio/templates/profile/schedule.tpl]: A form per single line?

File: view/theme/frio/templates/profile/schedule.tpl (hunk 2 above)
Comment (MrPetovan) [reply view/theme/frio/templates/profile/schedule.tpl]: We can polish the kinks in the theme stylesheets, please keep a single template.

Comment (annando) [reply view/theme/frio/templates/profile/schedule.tpl]: I just realized that the H-levels aren't consistent over the themes. I guess I will harmonize this. This is especially important for accessibility but also I dislike that chaos.

File: view/theme/frio/templates/profile/schedule.tpl [hunk 4]
```diff
@@ -0,0 +1,28 @@
+<div class="generic-page-wrapper">
+	{{* include the title template for the settings title *}}
+	{{include file="section_title.tpl" title=$title}}
+	<table id='application-block' class='table table-condensed table-striped'>
```
Comment (MrPetovan) [inline view/theme/frio/templates/profile/schedule.tpl]: Please use double-quotes for attribute values.
```suggestion
	<table id="application-block" class="table table-condensed table-striped">
```

File: view/templates/profile/schedule.tpl [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<div class="generic-page-wrapper">
+	<h1>{{$title}}</h1>
+	<table id='application-block' class='table table-condensed table-striped'>
```
Comment (MrPetovan) [inline view/templates/profile/schedule.tpl]:
```suggestion
	<table id="application-block" class="table table-condensed table-striped">
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: view/theme/frio/templates/profile/schedule.tpl (hunk 2 above)
Comment (annando) [reply view/theme/frio/templates/profile/schedule.tpl]: I just realized that the "delete" icon does only work with Frio, so we really need separate templates.

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**